docker-compose-generator/docker-fragments/opt-add-bwt.yml

```diff
@@ -0,0 +1,23 @@
+version: '3.4'
+
+services:
+  bwt:
+    image: shesek/bwt
+    environment:
+      network: mainnet
+      bitcoind-url: http://btcpayserver_bitcoind
+      xpub: <xpub1>,<xpub2>,<xpub3>
+      electrum-rpc-addr: 50001
+      http-server-addr: 3060
```

Comment:
need to expose through nginx

Reply:
Note that `electrum-rpc-addr` and `http-server-addr` should contain addresses, not just ports. For example, `127.0.0.1:3060` (or probably `0.0.0.0` in this case?).

Reply:
Thank you!  Still in draft mode, life is getting ahead of me...

Reply:
Yeah, use 0.0.0.0